### PR TITLE
make keepalive EnforcementPolicy.MinTime equals keepalive interval

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[x] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
fixes: #15088

File: galley/pkg/server/server.go [hunk 1]
```diff
@@ -175,7 +175,7 @@ func newServer(a *Args, p patchTable) (*Server, error) {
 		}),
 		// Relax keepalive enforcement policy requirements to avoid dropping connections due to too many pings.
 		grpc.KeepaliveEnforcementPolicy(keepalive.EnforcementPolicy{
-			MinTime:             time.Minute,
+			MinTime:             30 * time.Second,
```
Comment (howardjohn) [inline galley/pkg/server/server.go]: should this be `a.KeepAlive.Time`?

**Pull request merged**